### PR TITLE
feat: add dashboard for new synth metrics and refactor

### PR DESCRIPTION
**Pull request by zhongeric**
Follow up: 
- add more widgets and custom tracked pair widgets to the dashboard
- add alarms

### CODE REVIEW

File: lib/cron/synth-switch.ts [hunk 1]
```diff
@@ -260,10 +262,12 @@ async function main(metrics: MetricsLogger) {
           );
           try {
             await synthSwitchEntity.putSynthSwitch(SwitchRepository.parseKey(key), config.lowerBound[0], false);
+            metrics.putMetric(Metric.SYNTH_PAIR_DISABLED, 1, MetricLoggerUnit.Count);
+            metrics.putMetric(metricContext(Metric.SYNTH_PAIR_DISABLED, key), 1, MetricLoggerUnit.Count);
             return;
           } catch (e) {
             log.error({ key, error: e }, 'Failed to disable synthethics for trade');
-            metrics.putMetric(MetricName.SynthOrdersDisabledCount, 1, Unit.Count);
+            metrics.putMetric(metricContext(Metric.DYNAMO_REQUEST_ERROR, 'disable_synth'), 1, MetricLoggerUnit.Count);
```
Comment (rileydcampbell) [inline lib/cron/synth-switch.ts:270]: missing the context thing here?

File: lib/cron/synth-switch.ts [hunk 2]
```diff
@@ -394,12 +400,12 @@ async function main(metrics: MetricsLogger) {
         };
         return formattedRow;
       });
-      metrics.putMetric(MetricName.SynthOrdersCount, formattedResult.length, Unit.Count);
+      metrics.putMetric(Metric.SYNTH_ORDERS, formattedResult.length, MetricLoggerUnit.Count);
```
Comment (rileydcampbell) [inline lib/cron/synth-switch.ts:403]: Is this the total number of synth orders found?

**Review by rileydcampbell — COMMENTED** [review]
lgtm, just one question

**Review by rileydcampbell — APPROVED** [review]

File: lib/cron/synth-switch.ts (hunk 2 above)
Comment (zhongeric) [reply lib/cron/synth-switch.ts:403]: yea

File: lib/cron/synth-switch.ts (hunk 1 above)
Comment (zhongeric) [reply lib/cron/synth-switch.ts:270]: oo nice catch

File: lib/cron/synth-switch.ts [hunk 3]
```diff
@@ -245,6 +242,11 @@ async function main(metrics: MetricsLogger) {
           },
           'Outcome for trade'
         );
+        metrics.putMetric(Metric.SYNTH_ORDERS_POSITIVE_OUTCOME, pos, MetricLoggerUnit.Count);
+        metrics.putMetric(metricContext(Metric.SYNTH_ORDERS_POSITIVE_OUTCOME, key), pos, MetricLoggerUnit.Count);
```
[the remark this answers is not in the document]
Comment (zhongeric) [reply lib/cron/synth-switch.ts:246]: i thought it would be nice to be able to see overall positive orders

**Pull request merged**